static/src/javascripts/es6/projects/common/modules/crosswords/clues.js

```diff
@@ -41,6 +44,45 @@ class Clue extends React.Component {
 }
 
 export default class Clues extends React.Component {
+    constructor (props) {
+        super(props);
+        this.state = { showGradient: true };
+    }
+
+    componentDidMount () {
+        this.$cluesNode = React.findDOMNode(this.refs.clues);
+
+        const height = this.$cluesNode.scrollHeight - this.$cluesNode.clientHeight;
+
+        bean.on(this.$cluesNode, 'scroll', e => {
+            const showGradient = height - e.currentTarget.scrollTop > 20;
+
+            if (this.state.showGradient !== showGradient) {
+                this.setState({ showGradient: showGradient });
+            }
+        });
+    }
+
+    /**
+     * Scroll clues into view when they're activated (i.e. clicked in the grid)
+     */
+    componentDidUpdate (prev) {
+        if (!prev.focussed || prev.focussed.id !== this.props.focussed.id) {
+            this.scrollIntoView(this.props.focussed);
+        }
+    }
+
+    scrollIntoView (clue) {
+        const buffer = 100;
+        const node = React.findDOMNode(this.refs[clue.id]);
+        const visible = node.offsetTop - buffer > this.$cluesNode.scrollTop &&
```

Comment:
Should we wrap this in the ``detect.getBreakpoint() === 'tablet'`` as accessing offsetTop, scrollTop etc causes reflow and we can avoid doing that on narrow?